MCprep_addon/conf.py

```diff
@@ -137,28 +134,28 @@ def __init__(self):
 		# that no reading has occurred. If lib not found, will update to [].
 		# If ever changing the resource pack, should also reset to None.
 		self.material_sync_cache: List = []
-	
+
 		# Whether we use PO files directly or use the converted form
 		self.use_direct_i18n = False
 		# i18n using Python's gettext module
 		#
 		# This only runs if translations.py does not exist
+		self.languages: dict[str, gettext.NullTranslations] = {}
 		try:
 			if not self.translations.exists():
-				self.languages: dict[str, gettext.NullTranslations] = {}
 				for language in self.languages_folder.iterdir():
-					self.languages[language.name] = gettext.translation("mcprep", 
-											 self.languages_folder, 
-											 fallback=True,
-											 languages=[language.name])
+					self.languages[language.name] = gettext.translation(
+						"mcprep",
+						self.languages_folder,
+						fallback=True,
+						languages=[language.name])
 			self.use_direct_i18n = True
```

Comment:
This and the line underneath need to be indented by one more tab. I think #628 didn't properly indent these

Reply:
Do you mean from a wrong behavior or codestyle perspective? Hard to see on github, but it looks good to my eyes and aligns with pep8 (unintended 2x for clarify)

```python
if not self.translations.exists():
	for language in self.languages_folder.iterdir():
		self.languages[language.name] = gettext.translation(
			"mcprep",
			self.languages_folder,
			fallback=True,
			languages=[language.name])
self.use_direct_i18n = True
self.log("Loaded direct i18n!")
```

Reply:
It's wrong behavior. `self.use_direct_i18n` is what tells `MCprepEnv._` to use the MO files directly, and thus should be in the if condition.

(Last-night me forgot to move that over when adding the try-except block)